src/components/modalTabs/modalTabs.vue

```diff
@@ -94,14 +96,14 @@ export default {
     bottom: 0;
     left: 0;
     width: 0;
-    background-color: var(--primary);
+    background-color: var(--primary-button);
     transition: width .25s cubic-bezier(0, 1.61, 1, 1.23);
   }
 
   &[aria-selected='true'],
   &:hover,
   &:focus {
-    background-color: var(--background-color);
+    background-color: var(--modal-background-color);
```

Comment:
Would we want background colors like this tied to accents? To make theming easier?

Reply:
in favor of generalizing this, thoughts on a name? Maybe the background should be one of the extreme neutrals